index.html

```diff
@@ -27,110 +28,373 @@
       shape-rendering: crispEdges;
     }
 
+    .axis .minor line {
+      stroke: #777;
+      stroke-dasharray: 2,2;
+    }
+
     .axis text {
       font-size: 11px;
     }
 
     .label {
       font-weight: 700;
+
     }
 
     .point {
-      stroke: rgb(51, 122, 183);
-      stroke-width: 1;
-      fill: rgba(51, 122, 183, .5);
+      stroke: #FFFFFF;
+      stroke-width: 2;
+      fill: #358d88;
       opacity: .5;
     }
 
     .point.filtered {
       opacity: .05;
     }
 
-    .bar {
-      fill: rgb(51, 122, 183);
-    }
+
 
     .line {
       stroke: rgb(175, 80, 75);
       stroke-width: 2;
     }
+
+    body {
+      background-color: #5A455A;
+      font-family: "Lato", sans-serif;
+      
+    }
+
+    .category-label {
+      margin-right: 10px;
+      color: #ffffff;
+    }
+
+    .axis path,
+    .axis line {
+      fill: none;
+      stroke: #ffffff;
+      shape-rendering: crispEdges;
+    }
+
+
+
+    .axis text {
+      font-family: "Lato", sans-serif;
+      font-size: 11px;
+      fill: #ffffff;
+    }
+
+    p {
+      margin-top: 5px;
+    }
+
+    a {
+      font-size: 12px;
+      color: #ffffff;
```

Comment:
A small note here: Bootstrap, in addition to specifying the color of `a` elements, also has a rule for `a:focus, a:hover` that is applied when the link is focused (by a user navigating with the tab key) or moused over.

Unfortunately, the dark blue that Bootstrap uses is illegible against your purple background. So you probably want to override Bootstrap's `a:focus, a:hover` rule with one of your own, providing a better color.